### PR TITLE
[multistream]: Add typings for "multistream" package

### PR DESCRIPTION
**Pull request by mrmlnc**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/multistream/multistream-tests.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import * as fs from 'fs';
+import * as stream from 'stream';
+
+import * as through from 'through';
+import * as through2 from 'through2';
+import * as multistream from 'multistream';
```
Comment (sandersn) [inline types/multistream/multistream-tests.ts]: please use `import multistream = require('multistream')` for commonjs modules, which use `export = multistream` syntax.

**Review by sandersn — CHANGES_REQUESTED** [review]

File: types/multistream/multistream-tests.ts (hunk 1 above)
Comment (mrmlnc) [reply types/multistream/multistream-tests.ts]: Oh, sorry, I thought that this recommendation applies only to packages that return only one function. Now I understand that it's applies to all packages that returns function.

Comment (mrmlnc) [reply types/multistream/multistream-tests.ts]: Fixed.

**Comment by mrmlnc** [comment]
>  It appears Travis did not correctly run on this PR! 

Well, I rebase PR from master and Travis run build.

**Review by sandersn — APPROVED** [review]

**Pull request merged**